### ISSUE TITLE
Improving demo accessibility

### ISSUE THREAD

**Issue by amtins** [SRGSSR/pillarbox-web#151]
**As a Pillarbox developer**, I'd like the demo to be accessible via a screen reader.

## Acceptance Criteria

- Add appropriate narrated feedback to provide
  - information about the context
  - information about the state
- Disable keyboard navigation of DRM settings when not visible
